ontology/EBUCorePlus/ebucoreplus.owl

```diff
@@ -591,6 +591,7 @@ ec:hasAnimalColourCode rdf:type owl:ObjectProperty ;
 
 ###  http://www.ebu.ch/metadata/ontologies/ebucoreplus#hasAnimalGroom
 ec:hasAnimalGroom rdf:type owl:ObjectProperty ;
+                  owl:inverseOf ec:isAnimalGroomFor ;
```

Comment:
Should it be isAnimalGroomerFor and hasAnimalGroomer ?

Reply:
Oxford Learner's Dictionary says: "groom" is "a person whose job is to feed and take care of horses, especially by brushing and cleaning them". "groomer" is not found at all. Therefore, the proposed change is correct.